frontend/src/index.jade

```diff
@@ -26,7 +26,8 @@ html
               strong RepoSense
             span &nbsp; on {{ creationDate }}
 
-        #tabs-wrapper(v-if="isTabActive")
+        #tabs-wrapper(v-if="isTabActive", v-bind:style="{ flex: `0 0 ${flexWidth * 100}%` }")
+          .tab-resize(v-on:mousedown="registerMouseMove", v-on:mousemove="mouseMove", v-on:mouseup="deregisterMouseMove")
```

Comment:
same and why did you have to do it in two place tho? :thinking:

Reply:
At first I thought that without it, we can only register the mouse event outside of the resize bar